src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Hl7v2/OML_O21/MDHHS-OML-O21-1-expected.json

```diff
@@ -90,15 +138,15 @@
       }
     },
     {
-      "fullUrl": "urn:uuid:cec94dac-723d-33e4-3899-9e2b4a6c3be7",
+      "fullUrl": "urn:uuid:7bed24f6-d6fa-1218-ced1-b5d66ac0f024",
```

Comment:
Why is the ID of provenance changed in tests?

Reply:
Because the `Provenance` uses the `MSH.Value` for generating ID, and we added the `NormalizeText` recently, which changed the `MSH.Value`. Most other resources don' directly use the `MSH.Value`, so the IDs don't change.
Though ID changes, we can still say the `Provenance` resource is consistent.

Reply:
Got it. Thanks for the investigation!